client/blocks/inline-help/inline-help-search-card.jsx

```diff
@@ -71,9 +72,15 @@ class InlineHelpSearchCard extends Component {
 	onSearch = ( searchQuery ) => {
 		debug( 'search query received: ', searchQuery );
 		this.props.recordTracksEvent( 'calypso_inlinehelp_search', { search_query: searchQuery } );
+
+		// Make a search
 		this.props.requestInlineHelpSearchResults( searchQuery );
 	};
 
+	componentDidMount() {
```

Comment:
I _definitely_ prefer the Hooks API. This is a result of the copy/paste from existing components. That said, now we have fresh components it would be a minor refactor to move towards the more progressive API. Will do.

Reply:
Created [a followup Issue](https://github.com/Automattic/wp-calypso/issues/42699).